clusters/cijioagents2.yaml

```
@@ -12,7 +12,7 @@ releases:
   - name: datadog
     namespace: datadog
     chart: datadog/datadog
-    version: 3.84.2 #todo update updatecli/updatecli.d/charts/datadog.yaml
+    version: 3.84.3 #todo update updatecli/updatecli.d/charts/datadog.yaml
```

```suggestion
    version: 3.84.3
```

nit : forgot to remove the comment